packages/steiger/src/features/apply-globs.test.ts

```diff
@@ -0,0 +1,328 @@
+import { expect, it, describe } from 'vitest'
+
+import applyGlobs from './apply-globs'
+import { joinFromRoot, parseIntoFsdRoot } from '../../../steiger-plugin-fsd/src/_lib/prepare-test'
```

Comment:
Should we create a shared folder for test utilities for both Steiger and the fsd-plugin? Because, maybe it's just me, but I don't really like this import

Reply:
I think we can just copy-paste that file, no need to set up a whole separate folder for it

Reply:
Let me know if you'd like to do it before merging, because I'm honestly fine merging it as is